packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -178,16 +207,20 @@ export const codeChanged: Action<{
     return;
   }
 
-  if (state.live.isLive && !noLive) {
-    state.live.receivingCode = true;
-    effects.live.sendCodeUpdate(moduleShortid, module.code || '', code);
-    state.live.receivingCode = false;
+  if (state.live.isLive) {
+    effects.live.sendCodeUpdate(moduleShortid, module.code, event);
```

Comment:
Interesting that we did a `receivingCode` here before. I don't think it's needed while reading this code.

Reply:
Yeah, this is old stuff, I will remove it 😄